tutorials/speaker_tasks/End_to_end_Diarization_Inference.ipynb

```diff
@@ -0,0 +1,427 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell.\n",
+    "\n",
+    "## Install dependencies\n",
+    "!pip install wget\n",
+    "!apt-get install sox libsndfile1 ffmpeg\n",
+    "!pip install text-unidecode\n",
+    "!pip install ipython\n",
+    "\n",
+    "# ## Install NeMo\n",
+    "BRANCH = 'main'\n",
```

Comment:
Should we pin to a release version (or maybe add that as a comment) in case the notebook becomes inconsistent with future main?

Reply:
We change this part collectively when ever we update the release version.